src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        String firstBall = getRandomBall();
+        String secondBall = getRandomBall();
+        String thirdBall = getRandomBall();
```

Comment:
what if will be required to create 100500 balls? use loop instead
and you should pring balls to the console